docs/src/pages/system/properties/properties.md

```diff
@@ -1,60 +1,91 @@
-| [spacing](/system/spacing/)       | `spacing`        | `px,paddingX`      | `padding-left`, `padding-right`                                                              | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [spacing](/system/spacing/)       | `spacing`        | `py,paddingY`      | `padding-top`, `padding-bottom`                                                              | [`spacing`](/customization/default-theme/?expand-path=$.spacing)       |
-| [typography](/system/typography/) | `typography`     | `typography`       | `font-family`, `font-weight`, `font-size`, `line-height`, `letter-spacing`, `text-transform` | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `fontFamily`     | `fontFamily`       | `font-family`                                                                                | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `fontSize`       | `fontSize`         | `font-size`                                                                                  | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `fontWeight`     | `fontWeight`       | `font-weight`                                                                                | [`typography`](/customization/default-theme/?expand-path=$.typography) |
-| [typography](/system/typography/) | `textAlign`      | `textAlign`        | `text-align`                                                                                 | none                                                                   |
+<p class="description">This page lists all custom system properties, how are they linked with the theme, and which CSS properties are they resulting in.</p>
+
+For example, let's the the following row:
+
+| Group                       | System style function | System key        | CSS property/properties | Theme key                                                        |
+| :-------------------------- | :-------------------- | :---------------- | :---------------------- | :--------------------------------------------------------------- |
+| [spacing](/system/spacing/) | `spacing`             | `mb,marginBottom` | `margin-bottom`         | [`spacing`](/customization/default-theme/?expand-path=$.spacing) |
+
+1. The <b>Group</b> column gives you access to the documentation page where this group of properties are described, in the example, the [spacing](/system/spacing/) page.
+2. The <b>System style function</b> is referencing the style function which generates these properties, in case you want to use it for adding this functionality to your custom components. The function can be imported from `@material-ui/system`.
```

Comment:
Can we link to an example here how one would use it?

Reply:
Done with https://github.com/mui-org/material-ui/pull/23566/commits/50271358c1f3653bfd89cae83439644c3f26cbe4

Reply:
Shouldn't we have a dedicated page for that and not just some inlined example? How would I find this if I just want to use `@material-ui/system/properties` and not `sx` properties?

Reply:
We have started to cover this topic during Friday's meeting. There are a couple of aspects of the system that we do no longer document. These aspects could potentially lead to breaking changes or new documentation pages. From my perspective, there are two aspects:

1. Cherry-picking some sub-parts of the system. This is what this part of the documentation cover. Do we even need this in the first place?
2. Extensibility, how to had custom utils into the system?

In any case, I think that it goes beyond the scope of the pull request, but is definitely important to discuss to get a solid next step.

Reply:
We talked about adding Advanced page where we would dive into details of how the style function can be used. I believe that one could give more details. Should I add it as part of this PR so we can have the whole system updated, or would it be ok if I open new PR for that?

Reply:
If we export the system utilities we need to document them. Having this documentation living on an unrelated page is not really user-friendly.

Reply:
> If we export the system utilities we need to document them

We have two options, remove the exports or add the documentation. I think that what's best for v5 should influence the outcome, not the legacy.

> Having this documentation living on an unrelated page is not really user-friendly.

Yeah, I think that we could add a link to a dedicated page with more details.

Reply:
@eps1lon  I agree! @oliviertassinari according to our conversation, if we don't plan to advertise this functionality, we may drop this column. If not, I can open next PR for advanced usage and link these column to appropriate section there. I am ok with both options, leaning towards dropping the column as the `experimentalStyled` utility already supports adding the `sx` prop functionality on custom component. Adding additional style functions is already available by the `styled()` API so I am not even sure if this will really be used that much...

Reply:
Alright, will add as next PR the advanced page and will update this section. Not sure how much those will be used but it will be interesting to see anyway

Reply:
Sounds good. Leaving this conversation as unresolved so that we don't forget.